src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,60 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  hidden = false;
+
+  constructor(name, health = 100) {
+    super(name, health);
+
+    this.name = name;
+    this.health = health;
+
+    Animal.alive.push(this);
+  }
+
+  hide() {
+    return (this.hidden = true);
```

Comment:
[FUNCTION NAMING] - The `hide` method should only set the `hidden` property to `true`. Currently, it is returning the value of `this.hidden` after setting it. You should remove the return statement to comply with the task requirements.